backend/api/EventHandlers/MissionEventHandler.cs

```diff
@@ -159,8 +175,15 @@ private async void OnRobotAvailable(object? sender, RobotAvailableEventArgs e)
             }
 
             _scheduleMissionSemaphore.WaitOne();
-            try { await MissionScheduling.StartMissionRunIfSystemIsAvailable(missionRun.Id); }
-            catch (MissionRunNotFoundException) { return; }
+            try
```

Comment:
Consider keeping it inline rather than splitting over multiple lines

Reply:
done